distribution/docs/src/main/resources/_contents/_using/using-catalog-search-ui-contents.adoc

```diff
@@ -1,3 +1,216 @@
 
-// https://codice.atlassian.net/browse/DDF-2651
-placeholder text
+Introduction: The Catalog Search UI represents the most advanced search interface available with ${branding}.
+It provides a metadata search and discovery, resource retrieval and workspace management with a 3d or optional 2d map visualization.
+
+[NOTE]
+====
+For more detail on any feature or button with the Catalog UI, click the `?` button in the upper right of the screen.
+Hover over any item on the screen and a contextual tooltip will to define its purpose.
+To exit this mode, click the `?` again or press `escape`.
+====
+
+=== Accessing ${catalog-ui}
+
+The default location for the ${catalog-ui} is ${secure_url}/search/catalog
+
+=== Catalog UI Guest Users
+
+Guest users not signed in to ${branding} will have access to search functions, but all workspace configuration and settings will only exist locally and not be available for sharing.
+
+=== Workspaces in ${catalog-ui}
+
+Workspaces are collections of settings and standing searches that can be shared between users and stored to be accessed repeatedly.
```

Comment:
I _think_ workspaces are private/single user through the Standard Search UI, but have been opened up with this Catalog UI. I'll get some clarification today to make sure my assumption is still correct and update accordingly.